docs/build-on-polkadot/xcm/hrmp-channels/overview.md

```diff
@@ -0,0 +1,211 @@
+---
+title: HRMP Channels
+description: HRMP channels enable cross-chain communication in Polkadot, a temporary solution before the more efficient XCMP protocol is implemented.
```

Comment:
It seems that `index.md` description and `overview.md` description are the same, I think this would impact SEO